app/upgrades/v18/upgrades.go

```diff
@@ -26,6 +29,16 @@ func CreateUpgradeHandler(
 			return vm, err
 		}
 
+		expeditedPeriod := 24 * 7 * time.Hour // 7 days
+		govParams := keepers.GovKeeper.GetParams(ctx)
+		govParams.ExpeditedVotingPeriod = &expeditedPeriod
+		govParams.ExpeditedThreshold = govv1.DefaultExpeditedThreshold.String() // 66.7%
+		govParams.ExpeditedMinDeposit = govParams.MinDeposit                    // full deposit amount is required
```

Comment:
This needs to be 500 ATOMs. The min deposit is 250 ATOMs. See https://www.mintscan.io/cosmos/proposals/926

Reply:
Resolved in https://github.com/cosmos/gaia/pull/3149/commits/364db8496f98a7a31855730b3c0889834afe75e3